vue/components/ui/atoms/svg-viewer/svg-viewer.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <object class="svg-viewer" type="image/svg+xml" v-bind:data="svgData" />

```

Comment:
this should only render if `svgData` exists
```suggestion
    <object class="svg-viewer" type="image/svg+xml" v-if="svgData" v-bind:data="svgData" />

```